afs/templates/views/components/workflows/select-phys-thing-step.htm

```diff
@@ -18,8 +18,11 @@
     <div style="padding: 5px; margin-top: 15px;">
         <div class="widget-input-label">
             <span data-bind="if not: datasetRoute">
-                Select an Object
+                Select an Object from your project
             </span>
+            <div data-bind="if not: datasetRoute" style="font-style: italic; margin-bottom: 10px;">
+                Object must have at least one analysis area or sample location
+            </div>
             <span data-bind="if: datasetRoute == 'non-destructive'">
                 Select an Object that has an Area of interest
```

Comment:
I'm encountering a scenario where I still see the old label and the new label.  This seems incorrect.  
![Screenshot 2023-08-14 at 9 43 11 AM](https://github.com/archesproject/arches-for-science-prj/assets/124614/b07eae9c-022c-4bf5-8b36-d8b29986974c)

Reply:
Got some feedback from Cyrus, changes I just pushed should resolve this